test/traffictile.cc

```diff
@@ -112,7 +112,8 @@ TEST(Traffic, SpeedValid) {
   EXPECT_TRUE(speed.closed());
 
   // Test wraparound
-  speed.encoded_speed1 = UNKNOWN_TRAFFIC_SPEED_RAW + 1;
+  auto OVERFLOW_VALUE = UNKNOWN_TRAFFIC_SPEED_RAW + 1;
```

Comment:
It is the only way I found to make compiler stop complaining here 😄 But he is right: it seems we are doing bad thing here (but it is probably what we wanted to test).